### PR TITLE
Add the langchain-plotly integration

### PR DESCRIPTION
**Pull request by tpatel**

### CODE REVIEW

File: langchain-plotly/README.md
Comment (willydouhard) [inline langchain-plotly/README.md:1]: maybe give more information on how to edit the prompt and explain that this is educational content that does not perform well.

**Review by willydouhard — CHANGES_REQUESTED** [review]

File: langchain-plotly/README.md
Comment (tpatel) [reply langchain-plotly/README.md:1]: J'ai changé, tu en penses quoi?

**Review by willydouhard — APPROVED** [review]
let's wait for the release before merging

**Pull request merged**
